deploy/004-deploy-mock-pools.ts

```diff
@@ -65,30 +47,30 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   tx = await poolRegistry.createRegistryPool(
     "Pool 1",
+    proxyAdmin,
     Pool1Comptroller.address,
     closeFactor,
     liquidationIncentive,
     minLiquidatableCollateral,
     priceOracle.address,
   );
-
   await tx.wait();
 
   const pools = await poolRegistry.callStatic.getAllPools();
-  const comptroller1Proxy = await ethers.getContractAt("Comptroller", pools[0].comptroller);
+  const comptroller1Proxy = await ethers.getContractAt("Comptroller", pools[1].comptroller);
   tx = await comptroller1Proxy.acceptAdmin();
   await tx.wait();
-
-  const VBep20Immutable = await ethers.getContractFactory("VBep20Immutable");
-  const tokenImplementation = await VBep20Immutable.deploy();
-  await tokenImplementation.deployed();
+  
+  const VToken = await ethers.getContractFactory("VToken");
+  const vBNXImplementation = await VToken.deploy();
```

Comment:
Do we need the deployment args here?

Reply:
no

Reply:
How are we passing arguments like `name`?

Reply:
arguments are passed when calling `poolsRegistry.AddMarket`
this is the payload we are passing
`    struct AddMarketInput {
        address comptroller;
        address asset;
        uint8 decimals;
        string name;
        string symbol;
        InterestRateModels rateModel;
        uint256 baseRatePerYear;
        uint256 multiplierPerYear;
        uint256 jumpMultiplierPerYear;
        uint256 kink_;
        uint256 collateralFactor;
        uint256 liquidationThreshold;
        AccessControlManager accessControlManager;
        address vTokenProxyAdmin;
        VToken tokenImplementation_;
    }
`